packages/pn-commons/src/components/TermsOfServiceHandler.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Typography, Switch, Link, Button } from '@mui/material';
+import { useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { PRIVACY_LINK_RELATIVE_PATH, URL_DIGITAL_NOTIFICATIONS } from '../utils';
+
+type Props = {
+  handleAcceptTos: () => void;
+};
+
+const TermsOfServiceHandler = ({
+  handleAcceptTos,
+}: Props) => {
+  const { t } = useTranslation(['common']);
+  const [accepted, setAccepted] = useState(false);
+
+  const redirectPrivacyLink = () => window.location.assign(`${URL_DIGITAL_NOTIFICATIONS}${PRIVACY_LINK_RELATIVE_PATH}#privacy`);
+  const redirectToSLink = () => window.location.assign(`${URL_DIGITAL_NOTIFICATIONS}${PRIVACY_LINK_RELATIVE_PATH}#tos`);
+
+  return (
+    <Box>
+      <Typography mb={2} variant="h2" color="textPrimary" textAlign="center">
+        {t('tos.title')}
```

Comment:
Le localizzazioni in pn-commons funzionano in modo diverso rispetto gli altri packages, in quanto non c'è il modulo react-i18next (vedere package.json).
Per localizzare si usa il metodo getLocalizedOrDefaultLabel (vedere dove utilizzato da altre parti).